Makefile

```diff
@@ -396,7 +396,7 @@ lint-fix:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=0.13.4
+protoVer=0.13.5
```

Comment:
I'll be tagged proto-builder after this PR.
I added jq as it is useful to have in the container (in case we want to improve the protoc-gen-swagger script and need to read that config file).

Reply:
Building https://github.com/cosmos/cosmos-sdk/actions/runs/5592554942